polaris.shopify.com/content/components/link/index.md

```diff
@@ -49,6 +49,37 @@ The HTML that renders for the `Button` and `Link` components carries meaning. Us
 - a more cohesive visual experience for sighted users
 - products that are easier to maintain at scale
 
+### Open a new tab only when necessary
+
+Use default links whenever possible to avoid disorienting merchants and causing accessibility problems by opening a new tab.
+
+Links should be indicated as external when merchants are:
+
+- Performing a task or workflow, like creating a product
+- Navigating to a page outside of the Shopify admin
+
+### No icon
+
+Avoid using the [external link icon](/icons?icon=ExternalMinor&q=exter), as it can add unnecessary visual load inside a sentence or when accompanied by other content. Instead, add clarity to external links through clear link text and predictable placement of the link in a merchant’s workflow.
```

Comment:
```suggestion
Avoid using the [external icon](/icons?icon=ExternalMinor&q=external), as it can add unnecessary visual load inside a sentence or when accompanied by other content. Instead, add clarity to external links through clear link text and predictable placement of the link in a merchant’s workflow.
```